tools/devops/automation/templates/download-artifacts.yml

```diff
@@ -9,27 +14,29 @@ steps:
 - task: DownloadPipelineArtifact@2
   displayName: Download html report 
   inputs:
-    patterns: '**/HtmlReport.zip'
+    patterns: 'HtmlReport-${{ parameters.devicePrefix }}/HtmlReport.zip'
     allowFailedBuilds: true
     path: $(System.DefaultWorkingDirectory)/Reports
 
 # Unzip report.
 - task: ExtractFiles@1
   displayName: 'Extract HmlReport'
   inputs:
-    archiveFilePatterns: '$(System.DefaultWorkingDirectory)/Reports/HtmlReport/HtmlReport.zip'
-    destinationFolder: '$(System.DefaultWorkingDirectory)/HtmlReport'
+    archiveFilePatterns: '$(System.DefaultWorkingDirectory)/Reports/HtmlReport-${{ parameters.devicePrefix }}/HtmlReport.zip'
+    destinationFolder: '$(System.DefaultWorkingDirectory)/HtmlReport-${{ parameters.devicePrefix }}'
 
 # Download the test report to write the comment.
 - task: DownloadPipelineArtifact@2
   displayName: Download Test Summary
   inputs:
-    patterns: '**/TestSummary.md'
+    patterns: '**/TestSummary-${{ parameters.devicePrefix }}/TestSummary.md'
     allowFailedBuilds: true
     path: $(System.DefaultWorkingDirectory)\Reports
 
 - powershell: |
-    echo "##vso[task.setvariable variable=TEST_SUMMARY_PATH]$Env:SYSTEM_DEFAULTWORKINGDIRECTORY\Reports\TestSummary\TestSummary.md"
-    echo "##vso[task.setvariable variable=HTML_REPORT_PATH]$Env:SYSTEM_DEFAULTWORKINGDIRECTORY\HtmlReport"
+    Get-ChildItem -Recurse $Env:SYSTEM_DEFAULTWORKINGDIRECTORY
```

Comment:
I assume this is here for diagnostics. Just noting it in case you did not intend to leave it in. However, if generally useful for providing insights into the directory contents then certainly keep it.

Reply:
it is useful at the moment since we need to add some extra steps that extend this templates... so I want to be avle to see what is downloaded. Should delete later.